content/en/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

```diff
@@ -0,0 +1,206 @@
+but rather a host name, and the above command will have failed to set the `INGRESS_HOST` environment variable.
+Use the following command to correct the `INGRESS_HOST` value:
+
+{{< text bash >}}
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+{{< /text >}}
+
+{{< /warning >}}
+
+{{< /tab >}}
+
+{{< tab name="node port" category-value="node-port" >}}
+
+Follow these instructions if you have determined that your environment does not have an external load balancer,
+so you need to use a node port instead.
+
+Set the ingress ports:
+
+{{< text bash >}}
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+$ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
+{{< /text >}}
+
+Setting the ingress IP depends on the cluster provider:
+
+1.  _GKE:_
+
+    {{< text bash >}}
+    $ export INGRESS_HOST=<workerNodeAddress>
+    {{< /text >}}
+
+    You need to create firewall rules to allow the TCP traffic to the _ingressgateway_ service's ports.
+    Run the following commands to allow the traffic for the HTTP port, the secure port (HTTPS) or both:
+
+    {{< text bash >}}
+    $ gcloud compute firewall-rules create allow-gateway-http --allow tcp:$INGRESS_PORT
+    $ gcloud compute firewall-rules create allow-gateway-https --allow tcp:$SECURE_INGRESS_PORT
+    {{< /text >}}
+
+1.  _Minikube:_
+
+    {{< text bash >}}
+    $ export INGRESS_HOST=$(minikube ip)
+    {{< /text >}}
+
+1.  _Docker For Desktop:_
+
+    {{< text bash >}}
+    $ export INGRESS_HOST=127.0.0.1
+    {{< /text >}}
+
+1.  _Other environments (e.g., IBM Cloud Private etc):_
+
+    {{< text bash >}}
+    $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Configuring ingress using an Istio gateway
+
+A [Kubernetes Ingress Resources](https://kubernetes.io/docs/concepts/services-networking/ingress/) exposes HTTP and HTTPS routes from outside the cluster to services within the cluster.
+
+Let's see how you can configure a `Ingress` on port 80 for HTTP traffic.
+
+1.  Create an Istio `Gateway`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+      annotations:
+        kubernetes.io/ingress.class: istio
+      name: ingress
+    spec:
+      rules:
+      - host: httpbin.example.com
+        http:
+          paths:
+          - path: /status/*
+            backend:
+              serviceName: httpbin
+              servicePort: 8000
+    EOF
+    {{< /text >}}
+
+    The `kubernetes.io/ingress.class` annotation is required to tell Istio that it should handle this `Ingress`, otherwise it will be ignored.
```

Comment:
### 
```suggestion
    The `kubernetes.io/ingress.class` annotation is required to tell the Istio gateway controller that it should handle this `Ingress`, otherwise it will be ignored.
```